src/App/Pages/Accounts/LoginPasswordlessRequestPage.xaml.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.App.Models;
+using Bit.App.Utilities;
+using Xamarin.Forms;
+
+namespace Bit.App.Pages
+{
+    public partial class LoginPasswordlessRequestPage : BaseContentPage
+    {
+        private LoginPasswordlessRequestViewModel _vm;
+        private readonly AppOptions _appOptions;
+
+        public LoginPasswordlessRequestPage(string email, AppOptions appOptions = null)
+        {
+            InitializeComponent();
+            _appOptions = appOptions;
+            _vm = BindingContext as LoginPasswordlessRequestViewModel;
+            _vm.Page = this;
+            _vm.Email = email;
+            _vm.StartTwoFactorAction = () => Device.BeginInvokeOnMainThread(async () => await StartTwoFactorAsync());
+            _vm.LogInSuccessAction = () => Device.BeginInvokeOnMainThread(async () => await LogInSuccessAsync());
+            _vm.UpdateTempPasswordAction = () => Device.BeginInvokeOnMainThread(async () => await UpdateTempPasswordAsync());
+            _vm.CloseAction = async () => { await Navigation.PopModalAsync(); };
+
+            _vm.CreatePasswordlessLoginCommand.Execute(null);
+
+            ToolbarItems.Add(_closeItem);
```

Comment:
If the toolbar item is always added, then do it on xaml and move it away from the `ResourceDictionary` and therefore remove this line as well